res/mac/FreenetTray/src/LICENSE

```diff
@@ -1,81 +1,45 @@
 Original code:
 
 This program is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
+it under the terms of the GNU General Public License version 2, 
+as published by the Free Software Foundation.
```

Comment:
@bertm agreed :) This thing wouldn't work as a Mac App Store app (requires sandboxing which would break the ability to stop/start fred via run.sh), so it'll probably never be possible to put it on the MAS, and i wouldn't advise putting anything from FPI there anyway, too much loss of control over release timing and functionality :D

Anyhow the commit that changed this license file and the other headers has been cleanly reverted now :)